### PR TITLE
chore(message-system): zcash config

### LINKED ISSUE #5569: [Message] Update firmware to sign Zcash transaction
### First message
**Variant**: Warning (yellow)
**Dismissible**: True

**Conditions**

- Activated Zcash
- `AND` Suite app older then `22.6.2`

**Message**

- `en` _Please update your application to be able to sign Zcash transactions [Update Suite]_
- `es` _Por favor, actualiza tu aplicación para poder firmar transacciones de Zcash [Actualizar Suite]_
- `cs` _Aktualizujte prosím aplikaci, aby mohla podepisovat Zcash transakce [Aktualizovat Suite]_
- `ru` _Пожалуйста, обновите свое приложение, чтобы иметь возможность подписывать транзакции Zcash [Обновить Suite]
- `ja` _Zcashの取引に署名できるようにアプリケーションを更新してください [Suiteのアップデート]_

### Second message
**Variant**: Warning (yellow)
**Dismissible**: True

**Conditions**

- Activated Zcash
- `AND` Older firmware then actual one (`<1.11.1`, `<2.5.1`)
- `AND` Suite app version >=  `22.6.2`

**Message**
- `en` _Please update your firmware to be able to sign Zcash transactions [Update Firmware]_
- `es` _Para firmar transacciones de Zcash es necesario actualizar el firmware [Actualizar firmware]_
- `cs` _Aktualizujte prosím svůj firmware, abyste mohli podepisovat Zcash transakce [Aktualizovat Firmware]_
- `ru` _Для подписывания транзакций Zcach необходимо обновить прошивку [Обновить прошивку]
- `ja` _Zcashの取引に署名できるようにファームウェアをアップデートしてください。 [ファームウェアのアップデート]_

### PR DESCRIPTION
**Pull request by tomasklim**
closes #5569

### CODE REVIEW

**Comment by hynek-jina** [comment]
@tomasklim
I don't know how to test scenario with older Suite version.. 🤷‍♂️
And when I downgrade the firmware I receive message `New Trezor firmware is available! Please update your device.` And I can't close that message.. So that probably means that the second message is useless?

And please update the translations according this comment:
https://github.com/trezor/trezor-suite/issues/5569#issuecomment-1149963399

**Comment by tomasklim** [comment]
@hynek-jina you should see the second message in current develop web/desktop app

File: packages/suite-data/src/message-system/config/config.v1.json [hunk 1]
```diff
@@ -159,6 +159,111 @@
...
+                "id": "57934104-14e4-413f-8da1-7c7d42abe313",
+                "priority": 92,
+                "dismissible": true,
+                "variant": "warning",
+                "category": "banner",
+                "content": {
+                    "en-GB": "Please update your application to be able to sign Zcash transactions",
+                    "en": "Please update your application to be able to sign Zcash transactions",
+                    "es": "Por favor, actualiza tu aplicación para poder firmar transacciones de Zcash",
+                    "cs": "Aktualizujte prosím aplikaci, aby mohla podepisovat Zcash transakce",
```
Comment (matejkriz) [inline packages/suite-data/src/message-system/config/config.v1.json]:
```suggestion
                    "cs": "Aktualizujte prosím aplikaci, abyste mohli podepisovat Zcash transakce",
```

**Review by matejkriz — APPROVED** [review]
we should deploy this to production together with 22.6.2 public release

**Comment by matejkriz** [comment]
@hynek-jina "New Trezor firmware is available" banner is not dismissible, but has lower priority than this message. So when it's deployed you should see "Please update your application to be able to sign Zcash transactions" in older Suite version with older FW and ZEC activated.

**Pull request merged**